src/atoms/Button/index.js

```diff
@@ -0,0 +1,29 @@
+import classNames from "classnames"
+import PropTypes from "prop-types"
+import React, { Component } from "react"
+import styles from "./button.module.css"
+
+class Button extends Component {
```

Comment:
We can convert it to a Functional Component.

Reply:
Done.